src/framework/standard/help_commands.rs

```diff
@@ -1039,7 +1039,7 @@ fn flatten_group_to_string(
     };
 
     if summary_or_prefixes {
-        let _ = write!(group_text, "\n");
+        let _ = writeln!(group_text,);
```

Comment:
```suggestion
        let _ = writeln!(group_text);
```

Reply:
i thought the comma is required but yeah it's not.